framework/src/modules/chain/helpers/config.js

```diff
@@ -23,7 +23,7 @@ const configSchema = require('../schema/config');
 const { ZSchema } = require('../../../controller/helpers/validator');
 
 const validator = new ZSchema();
-const rootPath = path.dirname(path.resolve(__filename, '..'));
+const rootPath = path.dirname(path.resolve(__filename, '../../../../..'));
```

Comment:
since we already have root path, I prefer to use in the requires below, to make it more clear.